ci-operator/step-registry/aws/provision/iam-user/aws-provision-iam-user-commands.sh

```diff
@@ -93,5 +93,5 @@ aws_secret_access_key = ${key_sec}
 EOF
 
 # for destroy
-echo ${POLICY_ARN} >"${SHARED_DIR}/aws_policy_arns"
-echo ${USER_NAME} >"${SHARED_DIR}/aws_user_names"
+echo ${POLICY_ARN} >> "${SHARED_DIR}/aws_policy_arns"
```

Comment:
Shall we update `USER_CREDENTIALS_OUTPUT_FILENAME` and `USER_NAME` to avoid mini permission keyword, after all, in this step, it only provision iam user, the user's permission is determinate by `PERMISSIONS_POLICY_FILENAME`.